source/connectors/kafka-sink-properties.txt

```diff
@@ -0,0 +1,249 @@
+       | ``[ { "oldName":"key.fieldA", "newName":"field1" }, { "oldName":"value.xyz", "newName":"abc" } ]``
+       |
+       | **Default**: ``[]``
+       | **Accepted Values**: A valid JSON array
+
+   * - field.renamer.regexp
+     - string
+     - | An inline JSON array with objects describing regexp settings. For example:
+       | ``[ {"regexp":"^key\\\\..*my.*$", "pattern":"my", "replace":""}, {"regexp":"^value\\\\..*$", "pattern":"\\\\.", "replace":"_"} ]``
+       |
+       | **Default**: ``[]``
+       | **Accepted Values**: A valid JSON array
+
+   * - key.projection.list
+     - string
+     - | A comma-separated list of field names for key projection.
+       |
+       | **Default**: ``""``
+       | **Accepted Values**: A comma-separated list of field names
+
+   * - key.projection.type
+     - string
+     - | The type of key projection to use.
+       |
+       | **Default**: ``none``
+       | **Accepted Values**:  ``none``, ``blacklist``, or ``whitelist``
+
+   * - post.processor.chain
+     - list
+     - | A comma-separated list of post-processor classes to process the data before saving to MongoDB.
+       |
+       | **Default**: ``[ com.mongodb.kafka.connect.sink.processor.DocumentIdAdder ]``
+       | **Accepted Values**: A list of fully qualified Java class names
+
+   * - rate.limiting.every.n
+     - int
+     - | After how many processed batches the rate limit should trigger (NO rate limiting if n=0)
+       |
+       | **Default**: ``0``
+       | **Accepted Values**: An integer
+
+   * - rate.limiting.timeout
+     - int
+     - | How long in ms processing should wait before continuing processing.
+       |
+       | **Default**: ``0``
+       | **Accepted Values**: An integer
+
+   * - value.projection.list
+     - string
+     - | A comma-separated list of field names for value projection.
+       |
+       | **Default**: ``""``
+       | **Accepted Values**: A comma-separated list of field names
+
+   * - value.projection.type
+     - string
+     - | The type of value projection to use.
+       |
+       | **Default**: ``none``
+       | **Accepted Values**: ``none``, ``blacklist``, or ``whitelist``
+
+   * - writemodel.strategy
+     - string
+     - | The class that handles how build the write models for the sink documents.
+       |
+       | **Default**: ``com.mongodb.kafka.connect.sink.writemodel.strategy.ReplaceOneDefaultStrategy``
+       | **Accepted Values**: A fully qualified Java class name
+
+   * - topic.override.<topicName>.<propertyName>
+     - string
+     - | Specifies per-topic configuration overrides. For example:
+       | ``topic.override.foo.collection=bar`` instructs the connector sink to store data from the ``foo`` topic in the ``bar`` collection. See :ref:`topic-specific-configuration` for additional examples.
+
+       .. note::
+       
+          All configuration options can be specified on a per-topic basis
+          except ``connection.uri`` and ``topics``.
```

Comment:
nit: active -> passive

You can specify any configuration on a per-topic basis except for...

Reply:
Oops, my mistake. Will fix.